mono/mini/exceptions-amd64.c

```diff
@@ -108,7 +110,7 @@ get_win32_restore_stack (void)
 	amd64_mov_reg_imm (code, AMD64_R11, mono_restore_context);
 	amd64_call_reg (code, AMD64_R11);
 
-	g_assert ((code - start) < 128);
+	g_assertf ((code - start) <= size, "%d %d", (int)(code - start), size);
 
```

Comment:
Does this really fail often enough for this message to be useful ?

Reply:
Only if you try to enable `single_imm_size` globally as I tried.
Just `assert` but not `assertf`?
Or dismiss entire PR?